packages/stores/src/mockStore.ts

```diff
@@ -127,10 +127,12 @@ const mock = (): TStores & { is_mock: boolean } => {
             has_active_real_account: false,
             has_fiat: false,
             has_logged_out: false,
+            has_cookie_account: false,
```

Comment:
What's the purpose of the flag? Looks like a bit different name

Reply:
it came from here https://github.com/binary-com/deriv-app/pull/1606
in the store too, the function containing this variable is something named after cookie. this is used to set customer name and email for live chat. as per discussion with amina, lets not change this now.